### PR TITLE
[hi] Localize containerd.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/containerd.md` to Hindi `content/hi/docs/reference/glossary/containerd.md`

Fixes :- #35600

### CODE REVIEW

**Comment by bishal7679** [comment]
/assign @divya-mohan0209 @anubha-v-ardhan

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/containerd.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka:
+tags:
+  - tool
+---
+
+सादगी, मजबूती और सुवाह्यता पर जोर देने वाला कंटेनर रनटाइम
+
+<!--more-->
+
+कंटेनरडी एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} रनटाइम है जो Linux या Windows पर एक डेमॉन के रूप में चलता है। कंटेनरड फेचिंग का ख्याल रखता है और कंटेनर इमेजेसों को संग्रहीत करना, कंटेनरों को निष्पादित करना, नेटवर्क पहुंच प्रदान करना, और बहुत कुछ।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/containerd.md:18]:
```suggestion
कंटेनरडी एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} रनटाइम है जो Linux या Windows पर एक डैमन के रूप में चलता है। कंटेनरडी कंटेनर इमेजेस को लाने और संग्रहीत करने, कंटेनरों को निष्पादित करने, नेटवर्क एक्सेस प्रदान करने, आदि का ध्यान रखता है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Comment by divya-mohan0209** [comment]
@bishal7679 : PTAL whenever you have some free cycles.

**Comment by NitishKumar06** [comment]
@bishal7679 PING!

File: content/hi/docs/reference/glossary/containerd.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+aka:
+tags:
+  - tool
+---
+
+सादगी, मजबूती और सुवाह्यता पर जोर देने वाला कंटेनर रनटाइम
+
+<!--more-->
+
+कंटेनरडी एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} रनटाइम है जो Linux या Windows पर एक डैमन के रूप में चलता है। कंटेनरडी कंटेनर इमेजेस को लाने और संग्रहीत करने, कंटेनरों को निष्पादित करने, नेटवर्क एक्सेस प्रदान करने, आदि का ध्यान रखता है।
```
Comment (NitishKumar06) [inline content/hi/docs/reference/glossary/containerd.md:18]:
```suggestion
कंटेनरडी एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} रनटाइम है जो Linux या Windows पर एक डैमन के रूप में चलता है। कंटेनर डी कंटेनर इमेजेस को लाने और संग्रहीत करने, कंटेनरों को निष्पादित करने, नेटवर्क एक्सेस प्रदान करने, आदि का ध्यान रखता है।
```
Native Hindi speakers might pronounce it in a different way (containerd) , hence suggesting this. Your thoughts about it?

**Review by NitishKumar06 — CHANGES_REQUESTED** [review]
Overall looks good. However, please put your thoughts on the suggestion raised.

File: content/hi/docs/reference/glossary/containerd.md (hunk 2 above)
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/containerd.md:18]: I don't think the spacing makes a difference. @bishal7679 : Please weigh in here.

**Comment by bishal7679** [comment]
I didn't get the reason why deploying/building getting failed again & again :(
So raising another new PR via taking ref from this and closing this one!
/close

**Pull request closed without merging**